src/js/models/v2/awards/BaseContract.js

```diff
@@ -81,15 +81,15 @@ BaseContract.populate = function populate(data) {
         this.additionalDetails = additionalDetails;
     }
 
-    this.parentAward = data.parent_award_piid || '--';
+    this.parentAward = data.parent_award_id || '--';
```

Comment:
We need to look at **both** the `additionalDetails.js` mapping and the original `AwardSummary.js` model object. In the original logic, the container created an `AwardSummary` object from the API response, which remapped some API fields to custom properties. Where fields were not remapped, the original API field would be appended to the object. The presentation component would then **re-remap** those fields again via the `additionalDetails.js` file. So when we are updating the code to be simpler (and have only one remapping), we need to consult `additionalDetails` first, then trace back to `AwardSummary` to see if it is a double-remapped field or if it is an original API field.

In this case, the `parent_award_id` field in `additionalDetails.js` is pointing to a field that was remapped by `AwardSummary` from the API's `parent_award_piid` field.

From:
https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/models/results/award/AwardSummary.js#L223
to:
https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/models/results/award/AwardSummary.js#L478